docs/proposals/podpolicy.md

```diff
@@ -0,0 +1,149 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/choosing-scheduler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# PodPolicy and its use in choosing the scheduler in a multi-scheduler system
+
+## Introduction
+
+This document describes a new API resource, `PodPolicy`, for defining policies that apply to pods
+based on the pods' characteristics. Its initial use is to serve as the configuration for a
+new admission controller that will assign the scheduler name annotation in a
+[multi-scheduler](multiple-schedulers.md) Kubernetes system.
+
+## API
+
+```go
+// PodPolicySpec defines the configuration of the PodPolicy API resource.
+// Each rule is tested in order, and the first one that matches is applied.
```

Comment:
I would expect that different kinds of policy might have different selectors, making this "first match" awkward.  But maybe I misunderstand how you think this is to be used?  Assume I want to set a color-based annotation and a network perfomance policy.  Is it 

a) One PodPolicy per namespace
```
kind: PodPolicy
metadata:
  name: one-true-policy
spec:
  rules:
  - predicate:
      color: blue
    annotations:
      temperature: cool
    network: slow
  - predicate:
      color: red
    annotations:
      temperature: warm
    network: fast
  - annotations:
      temperature: neutral
    network: fast
```

This works as long as my various policies are using the same predicates (unlikely).  Or is it:

```
kind: PodPolicy
metadata:
  name: color-policy
spec:
  rules:
  - predicate:
      color: blue
    annotations:
      temperature: cool
  - predicate:
      color: red
    annotations:
      temperature: warm
  - annotations:
      temperature: neutral
```

```
kind: PodPolicy
metadata:
  name: network-policy
spec:
  rules:
  - predicate:
      department: eng
    network: fast
  - predicate:
      department: marketing
    network: slow
  - network: medium
```